packages/docregistry/src/registry.ts

```diff
@@ -861,6 +866,8 @@ export namespace DocumentRegistry {
     addSibling(widget: Widget, options?: IOpenOptions): IDisposable;
   }
 
+  export type SaveState = 'starting' | 'completed' | 'failed';
```

Comment:
How do you feel about making the tenses match?
```typescript
export type SaveState = 'started' | 'completed' | 'failed';
```

Reply:
I could imagine an argument being made for special-casing 'starting' ... but my gut feeling is to make them consistent.

Reply:
Sounds good, I'll make the change